packages/chart/src/types/ChartConfig.ts

```diff
@@ -188,8 +189,10 @@ export type ForestPlotConfig = {
 } & AllChartsConfig
 
 export type LineChartConfig = {
-  visualizationType: 'Line'
+  convertLineToBarGraph: boolean
+  displayLineGraphAs: 'Line'
```

Comment:
why do you need both of these?

Reply:
The convertLineToBarGraph is the checkbox value that is giving permission to change the line to a bar. Should I rename it to allowLineToBeConvertedToBar or something like that? Or do you believe I can get around not using it at all?